model/pod/podutil.go

```diff
@@ -9,40 +9,29 @@ import (
 	"github.com/pkg/errors"
 )
 
-const (
-	executable = "./evergreen"
-	shell      = "CMD-SHELL "
-)
-
 // Constants representing default curl retry arguments.
 const (
 	curlDefaultNumRetries = 10
 	curlDefaultMaxSecs    = 100
 )
 
-// CurlCommandWithRetry returns the command to curl the evergreen client and retries the request.
-func (p *Pod) CurlCommandWithRetry(settings *evergreen.Settings, numRetries, maxRetrySecs int) (string, error) {
+// CurlCommand returns the command to curl the evergreen client and retries the request with the default retry parameters.
+func (p *Pod) CurlCommand(settings *evergreen.Settings) ([]string, error) {
 	var retryArgs string
-	if numRetries != 0 && maxRetrySecs != 0 {
-		retryArgs = " " + curlRetryArgs(numRetries, maxRetrySecs)
+	if curlDefaultNumRetries != 0 && curlDefaultMaxSecs != 0 {
```

Comment:
I think we don't need this check since we're not passing in parameter to `CurlCommand` and the constants are already non-zero - you can just construct the `curlRetryArgs` without this if check:
```go
retryArgs := " " + curlRetryArgs(curlDefaultNumRetries, curlDefaultMaxSecs)
```